src/Tribe/Tickets.php

```diff
@@ -1,2299 +1,2352 @@
+	/**
+	 * Retrieve the ID numbers of all tickets of an event
+	 *
+	 * @since  TBD
+	 *
+	 * @param  int|WP_Post $post
+	 *
+	 * @return array
+	 */
+	public function get_tickets_ids( $post ) {
+		if ( ! $post instanceof WP_Post ) {
+			$post = get_post( $post );
+		}
 
+		if ( ! $post instanceof WP_Post ) {
+			return false;
 		}
 
-		/**
-		 * Returns link to the report interface for sales for a single ticket or
-		 * null if the provider doesn't have reporting capabilities.
-		 * As of TBD we reversed the params and deprecated $post_id as it was never used
-		 *
-		 * @abstract
-		 *
-		 * @param deprecated $post_id ID of parent "event" post
-		 * @param int $ticket_id ID of ticket post
-		 * @return mixed
-		 */
-		public function get_ticket_reports_link( $post_id_deprecated, $ticket_id ) {
+		$args = $this->get_tickets_query_args( $post->ID );
+		$query = new WP_Query( $args );
 
-		}
+		return $query->posts;
+	}
 
-		/**
-		 * Returns a single ticket
-		 *
-		 * @abstract
-		 *
-		 * @param int $post_id ID of parent "event" post
-		 * @param int $ticket_id ID of ticket post
-		 * @return mixed
-		 */
-		public function get_ticket( $post_id, $ticket_id ) {
 
+	/**
+	 * Returns the html for the delete ticket link
+	 *
+	 * @since TBD
+	 *
+	 * @param object $ticket Ticket object
+	 * @return string HTMl link
+	 */
+	public function get_ticket_delete_link( $ticket = null ) {
+		if ( empty( $ticket ) ) {
+			return;
 		}
 
+		$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) ? __( 'Delete RSVP', 'event-tickets' ) : __( 'Delete Ticket', 'event-tickets' ) ;
+
 		/**
-		 * Returns the html for the delete ticket link
+		 * Allows for the filtering and testing if a user can delete tickets
 		 *
 		 * @since TBD
 		 *
-		 * @param object $ticket Ticket object
-		 * @return string HTMl link
+		 * @param bool true
+		 * @param int ticket post ID
+		 * @return string HTML link | void HTML link
 		 */
-		public function get_ticket_delete_link( $ticket = null ) {
-			if ( empty( $ticket ) ) {
-				return;
-			}
-
-			$button_text = ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) ? __( 'Delete RSVP', 'event-tickets' ) : __( 'Delete Ticket', 'event-tickets' ) ;
-
-			/**
-			 * Allows for the filtering and testing if a user can delete tickets
-			 *
-			 * @since TBD
-			 *
-			 * @param bool true
-			 * @param int ticket post ID
-			 * @return string HTML link | void HTML link
-			 */
-			if ( apply_filters( 'tribe_tickets_current_user_can_delete_ticket', true, $ticket->ID, $ticket->provider_class ) ) {
-				$delete_link = sprintf(
-					'<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" id="ticket_delete_%2$s" class="ticket_delete">%3$s</a></span>',
-					$ticket->provider_class,
-					$ticket->ID,
-					esc_html( $button_text )
-				);
-
-				return $delete_link;
-			}
-
+		if ( apply_filters( 'tribe_tickets_current_user_can_delete_ticket', true, $ticket->ID, $ticket->provider_class ) ) {
```

Comment:
add docblock for this filter and, if it's new, ensure it's mentioned in the readme.txt as a new filter

Reply:
That isn't new. I just removed indentation: `?w=1` on the changes